config/crd/bases/packages.eks.amazonaws.com_packagebundles.yaml

```diff
@@ -76,12 +76,33 @@ spec:
                                 description: Digest is a checksum value identifying
                                   the version of the package and its contents.
                                 type: string
+                              images:
+                                description: Images is a list of images used by this
+                                  version of the package
+                                items:
+                                  description: SourceVersion describes a version of
+                                    an package within a repository.
+                                  properties:
+                                    digest:
+                                      description: Digest is a checksum value identifying
+                                        the version of the package and its contents.
+                                      type: string
+                                    repository:
+                                      description: Repository is source in the registry
+                                        for the image
+                                      type: string
+                                  required:
+                                  - digest
+                                  - repository
```

Comment:
I think requiring repository makes sense but we have to re-organize much of what we currently have since we're sharing repositories for multiple images as well as the helm chart. We need to validate how harbor and ecr deal with nested repositories and settle on a standard, i.e. `registry/harbor/helm:1.0.0` + `registry/harbor/images/harbor:1.0.0`

Reply:
If we need to push helm charts to their own repositories, that is fine, but that is not a topic for this PR

Reply:
We filter by OCI Image Media Types currently when using the same repo for bundle creation. (Helm vs Image) so this should work just fine for that.